src/darkslide/cli.py

```diff
@@ -64,6 +64,13 @@ def _parse_options():
     )
 
     parser.add_option(
+        "-m", "--max-toc-level",
+        type="int",
+        dest="maxtoclevel",
+        help="Limits the TOC level generation to a specific level.",
+        default=9)
```

Comment:
Maybe change the default back to 2? Or 3?

Reply:
yes :)